Library/Formula/libharu.rb

```diff
@@ -13,22 +11,42 @@ class Libharu < Formula
     sha1 "0a20c683544726d870f53c005f79a730b1effffe" => :mountain_lion
   end
 
-  depends_on 'libpng'
-
-  # Fixes compilation against LibPNG 1.5. Can be removed on next release.
-  # Based on a commit in the LibHaru repository which does not apply cleanly
-  # due to a missing CHANGES file:
-  # https://github.com/tony2001/libharu/commit/e5bf8b0.patch
-  patch do
-    url "http://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/media-libs/libharu/files/libharu-2.2.1-libpng-1.5.patch?revision=1.1"
-    sha1 "463641d4570d59d632c1878597253db962129599"
+  head do
```

Comment:
Nitpick, but since the `head` block contains only a `url`, this can be written `head "https://github.com/libharu/libharu.git"`